polkadot-parachains/integritee-polkadot/src/xcm_config.rs

```diff
@@ -318,7 +317,7 @@ pub type Traders = (
 	// for DOT aka RelayNative
 	FixedRateOfFungible<
 		RelayNativePerSecond,
-		XcmFeesTo32ByteAccount<LocalFungiblesTransactor, AccountId, TreasuryAccount>,
+		XcmFeesTo32ByteAccount<ReservedFungiblesTransactor, AccountId, TreasuryAccount>,
```

Comment:
We failed to deposit the relay token into the treasury account without this. However, we will swap assets soon anyhow, and then this case shouldn't occur anymore.